masterlist.yaml

```
@@ -1049,6 +1049,27 @@ plugins:
   - name: 'Navmesh Fixes and Improvements.esm'
     url: [ 'https://www.nexusmods.com/newvegas/mods/62041/' ]
     group: *fixesGroup
+    msg:
+      - <<: *alreadyInX
+        subs: [ '**NavmeshOverhaul.esm**' ]
+        condition: 'active("NavmeshOverhaul.esm")'
+
+  - name: 'NavmeshOverhaul.esm'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/81003' ]
+    group: *fixesGroup
+
+  - name: 'YUPNavmeshPatch.esm'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/81003' ]
+    group: *fixesGroup
+    inc:
+      - name: 'Uncut Wasteland.esm'
+        detail: &checkScript "It is incompatible according to the author's error check script. Follow [the guide](https://www.nexusmods.com/newvegas/mods/81003)."
+      - name: "Vanilla Enhancements.esm"
+        detail: *checkScript
+      - name: "TLD_Travelers.esm"
+        detail: *checkScript
+      - name: "Functional Post Game Ending.esm"
+        detail: *checkScript
```

The inc can be replaced with `&compatIssuesWithX`.
> This plugin has compatibility issues with {0}. For more information, read this mod''s compatibility notes.

```suggestion
    msg:
      - <<: *compatIssuesWithX
        subs: [ 'Essential Vanilla Enhancements Merged' ]
        condition: 'active("Vanilla Enhancements.esm")'
      - <<: *compatIssuesWithX
        subs: [ 'Functional Post Game Ending' ]
        condition: 'active("Functional Post Game Ending.esm")'
      - <<: *compatIssuesWithX
        subs: [ 'The Living Desert - Travelers Patrols Consequences' ]
        condition: 'active("TLD_Travelers.esm")'
      - <<: *compatIssuesWithX
        subs: [ 'Uncut Wasteland' ]
        condition: 'active("Uncut Wasteland.esm")'
```